contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/HdfsCopyStats.java

```diff
@@ -151,6 +159,8 @@ public HdfsCopyStats(String source,
         this.lastReportNs = System.nanoTime();
         this.startTimeMS = System.currentTimeMillis();
         initializeStatsFile(destination, enableStatsFile, maxVersionsStatsFile, isFileCopy);
+
+        aggStats = HdfsFetcherAggStats.getStats();
```

Comment:
I would like to remove the method getStats and would like to make the aggStats static.